yaml_checker/README.md

```diff
@@ -0,0 +1,111 @@
+# YAML Checker
+
+An internal CLI util for formatting and validating YAML files. This project
+relies on Pydantic and Ruamel libraries.
+
+**Installation**
+```bash
+pip install -e yaml_checker
+```
+
+**Usage**
+```
+usage: yaml_checker [-h] [-v] [-w] [--config CONFIG] [files ...]
+
+positional arguments:
+  files            Additional files to process (optional).
+
+options:
+  -h, --help       show this help message and exit
+  -v, --verbose    Enable verbose output.
+  -w, --write      Write yaml output to disk.
+  --config CONFIG  CheckYAML subclass to load
+```
+
+**Example**
+
+```bash
+# Lets cat a demonstration file for comparison. 
+$ cat yaml_checker/demo/slice.yaml
```

Comment:
Let's have this as a `bash` block and the following yaml as a `yaml` block for better syntax.